types/koa__router/index.d.ts

```diff
@@ -386,8 +385,7 @@ declare class Router<StateT = Koa.DefaultState, ContextT = Koa.DefaultContext> {
      * // => "/users/3?limit=1"
      *
      */
-    url(name: string, params?: any, options?: Router.UrlOptionsQuery): string;
-    url(name: string, params?: any, options?: Router.UrlOptionsQuery): Error;
+    url(name: string, params?: any, options?: Router.UrlOptionsQuery): Error | string;
```

Comment:
Looking at the source code, this is actually plain wrong: this function will either _throw_ an `Error` or return a `string` - it won't return an `Error`.

I'll fix this in a follow-up PR